content/docs/getting-started/examples/N2.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Example on computing ionization potential and electron affinity from self-consistent GW for Nitrogen molecule
+linkTitle: IP and EA for molecule (N2)
+weight: 2
+icon: N2
+prev: "/docs/getting-started/postprocessing"
+---
+
+Here we provide a simple example on how to run `Green`/`Weakcoupling` for computing ionization potential (IP) for nitrogen molecule (Geometry obtained for GW100 dataset).
+Prerequisites: you must have the mbpt compiled and the [green-mbtools](https://pypi.org/project/green-mbtools) python package installed.
+Change to a new directory where you will keep your simulations, create a directory for the H2O simulation.
```

Comment:
I think here should be N2 instead of H2O
```suggestion
Change to a new directory where you will keep your simulations, create a directory for the $N_2$ simulation.
```

Reply:
Yes! Thanks for catching that.